allure-generator/src/main/javascript/components/tooltip/TooltipView.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Isn't that so hard to clean up unused values instead. Idea even highlights them.